db.php

```diff
@@ -1157,10 +1167,7 @@ public function db_version( $dbh_or_table = false ) {
 			$dbh = $this->db_connect( "SELECT FROM $dbh_or_table $this->users" );
 		}
 
-		if ( $dbh ) {
-			return preg_replace( '/[^0-9.].*/', '', $this->ex_mysql_get_server_info( $dbh ) );
-		}
-		return false;
+		return $dbh ? $this->ex_mysql_get_server_info( $dbh ) : false;
```

Comment:
Ideally we could remove the `ex_mysql_get_server_info()` method and do the logic here. But it's a public method so I wasn't sure. Should we maybe deprecate instead?